deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -746,7 +746,11 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
         if message.isInfo {
             let cell = tableView.dequeueReusableCell(withIdentifier: "info", for: indexPath) as? InfoMessageCell ?? InfoMessageCell()
             cell.showSelectionBackground(tableView.isEditing)
-            cell.update(text: message.text)
+            if let parent = message.parent, parent.type == DC_MSG_WEBXDC {
```

Comment:
with core101, this can be done more effective using `message.infoType == DC_INFO_WEBXDC_INFO_MESSAGE` (`.parent` requires a database call `.infoType` does not).

but that can go to subsequent pr when, there are probably also other places where this can be optimized.